.circleci/config.yml

```diff
@@ -24,34 +24,31 @@ commands:
         only_for_branches: master
         webhook: $SLACK_GREENKEEPER_WEBHOOK
   notify-rc-tag:
-    description: 'Notifies #eng-magento1 when a release candidate is tagged.'
+    description: 'Notify #eng-magento1 after a release candidate is tagged.'
     parameters:
       tag:
         type: string
         default: ''
     steps:
     - slack/notify:
-        channel: "#eng-magento1"
-        color: '#33ff00'
-        mentions: '@oleksii,@Pavel,'
+        color: '#58a359'
+        mentions: 'QAEng'
         message: ":white_check_mark: A new RC << parameters.tag >> has been tagged for Magento 1!"
-        webhook: $SLACK_GREENKEEPER_WEBHOOK
+        webhook: $SLACK_MAGENTO1_WEBHOOK
+        include_job_number_field: false
+        include_project_field: false
+        include_visit_job_action: false
 jobs:
-  tag-rc-candidate:
-    description: Tag master as rc every 3 weeks
+  tag-release-candidate:
     docker:
     - image: cimg/base:2020.01
     steps:
+    - checkout
     - run:
-        name: Validate eligibility to tag rc
-        command: .circleci/scripts/rc-tag.sh
-    - when:
-        condition: $TAGTHISWEEK
-        steps:
-        - swissknife/publish_github_release:
-            tag: $NEWTAGNAME
-        - notify-rc-tag:
-            tag: $NEWTAGNAME
+        name: Tag master as release candidate every 3 weeks
+        command: .circleci/scripts/tag-rc-if-necessary.sh
+    - notify-rc-tag:
+        tag: $NEWTAGNAME
```

Comment:
@adentong Have you tested that $NEWTAGNAME is passed between steps?
When I tested with M2 I needed
```
source .circleci/scripts/rc-tag.sh
echo "export NEWTAGNAME=$NEWTAGNAME" >> $BASH_ENV
```

Reply:
I moved that step to inside the script. And I tested that it's passed to the next step correctly.